index-01.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
We don't have to name an html file with numbers, so you can safely change this one to `index.html`.